src/main/java/de/srendi/advancedperipherals/common/items/MemoryCardItem.java

```diff
@@ -16,6 +19,8 @@
 
 public class MemoryCardItem extends BaseItem {
 
+    public static final String OWNER_NBT_KEY = "owner";
```

Comment:
we use `ownerId` but not `owner`
```suggestion
    public static final String OWNER_NBT_KEY = "ownerId";
```